### PR TITLE
Local windows running

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

**Comment by pallenSL** [comment]
:shipit:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Sessions
+===
+The sessions service only relies on a connection to the mysql database. 
+
+# Running with on Linux / git bash on windows
```
Comment (niksoper) [inline README.md]: Typo?  "with on Linux" should be "on Linux"?

Comment (jebbinSCL) [inline README.md]: Corrected

**Comment by niksoper** [comment]
:+1:

**Pull request merged**